src/components/PatternList.js

```diff
@@ -0,0 +1,64 @@
+import Icon from "react-native-vector-icons/FontAwesome";
+import { List } from "react-native-paper";
+import React from "react";
+import ScreenLayout from "../shared/ScreenLayout";
+import styled from "styled-components/native";
+
+const pageTitle = "Choose your patterns";
+const btnLabel = "次行くで";
+const patterns = [
+  "いる / おる　おるん?",
+  "よ / なんよ",
+  "ない / ん",
+  "しないで / しんさんな、せんといて",
+  "だ / じゃ",
+  "だろう / じゃろう",
+];
+
+export const PatternList = ({ navigation }) => {
+  return (
+    <ScreenLayout
+      pageTitle={pageTitle}
+      btnLabel={btnLabel}
+      backComponentName={"ChooseDialect"}
+      onPressHandler={() => navigation.navigate("Lesson")}
+    >
+      <PatternContainer>
+        {patterns.map((pattern, index) => {
+          return (
+            <ListTouchable key={index} onPress={() => console.log(pattern)}>
```

Comment:
Do you want the on press to be `() => navigation.navigate("Lesson")` tachimachi de?